CefSharp/IWebBrowser.cs

```diff
@@ -187,6 +187,12 @@ public interface IWebBrowser : IDisposable
         /// <value>The find handler.</value>
         IFindHandler FindHandler { get; set; }
 
+        /// <summary>
+        /// Implement <see cref="IAccessibilityHandler" /> to handle events related to accessibility.
+        /// </summary>
+        /// <value>The accessibility handler.</value>
+        IAccessibilityHandler AccessibilityHandler { get; set; }
```

Comment:
Remove this as it's only valid for `WPF` and `OffScreen`

Reply:
Should I move AccessibilityHandler to IRenderWebBrowser?

Reply:
Hmm, probably not. It's only implemented explicitly.

Shouldn't make much difference if you don't add it to an interface.

Reply:
But if I move OnAccessibilityLocationChange and OnAccessibilityTreeChange implementations to RenderClientAdapter.h then IRenderWebBrowser should have reference to AccessibilityHandler, right? RenderClientAdapter references IRenderWebBrowser.

Reply:
It does, my mistake. Just don't implement the property explicitly.

Reply:
Not sure if I got it correctly, but I moved CefAccessibilityHandler implementation to RenderClientAdapter and added IAccessibilityHandler to IRenderWebBrowser. Looks like it makes sense since we only need IAccessibilityHandler for WPF and OffScreen.